README.md

```diff
@@ -1,7 +1,97 @@
-# To add a new driver:
+# To add a new driver
 
 - Edit `versions.sh`, add new driver version.
 - Run `update-data.sh`
 
 The script will download the drivers from the nvidia servers and place the
  corresponding files in the `data` directory.
+
+# Building locally
+
+**1\.** Install the following packages:
+
+```bash
+flatpak-builder git make
+```
+
+**2\.** Add the Flathub repository:
+
+```bash
+flatpak --user remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+```
+
+**3\.** Clone this repository:
+
+```bash
+git clone https://github.com/flathub/org.freedesktop.Platform.GL.nvidia.git
+cd org.freedesktop.Platform.GL.nvidia
+```
+
+**4\.** Assuming you already know what driver version you want to build, use the following command to modify `versions.sh`:
+
+```bash
+# In this example, only the 560.35.03 driver will be built.
+# You can also build multiple versions by separating them with a space.
+echo 'DRIVER_VERSIONS="560.35.03"' >> versions.sh
+```
+
+5\. Install the appropriate 1.6 Freedesktop Platform/SDK for your CPU architecture:
+
+```bash
+flatpak --user install --no-related flathub "org.freedesktop.Platform/$(flatpak --default-arch)/1.6"
+flatpak --user install --no-related flathub "org.freedesktop.Sdk/$(flatpak --default-arch)/1.6"
```

Comment:
I usually prefer recommending they just add `--install-deps-from=flathub` to their f-b args.

Reply:
Yeah, sure, but "unfortunately" that ends up also installing more packages than we actually need, namely ffmpeg and video drivers (when they're not already installed).

That's why I opted to installing the Sdk and Platform packages manually; That way we only install what's *actually* needed...